doc/sphinx/Pacemaker_Explained/rules.rst

```diff
@@ -6,221 +6,102 @@
-   |                 |             | with a ``role`` of ``Promoted`` cannot    |
-   |                 |             | determine the initial location of a clone |
-   |                 |             | instance and will only affect which of    |
-   |                 |             | the active instances will be promoted.    |
-   +-----------------+-------------+-------------------------------------------+
-   | score           |             | .. index::                                |
-   |                 |             |    pair: rule; score                      |
-   |                 |             |                                           |
-   |                 |             | If this rule is used in a location        |
-   |                 |             | constraint and evaluates to true, apply   |
-   |                 |             | this score to the constraint. Only one of |
-   |                 |             | ``score`` and ``score-attribute`` may be  |
-   |                 |             | used.                                     |
-   +-----------------+-------------+-------------------------------------------+
-   | score-attribute |             | .. index::                                |
-   |                 |             |    pair: rule; score-attribute            |
-   |                 |             |                                           |
-   |                 |             | If this rule is used in a location        |
-   |                 |             | constraint and evaluates to true, use the |
-   |                 |             | value of this node attribute as the score |
-   |                 |             | to apply to the constraint. Only one of   |
-   |                 |             | ``score`` and ``score-attribute`` may be  |
-   |                 |             | used.                                     |
-   +-----------------+-------------+-------------------------------------------+
-   | boolean-op      | ``and``     | .. index::                                |
-   |                 |             |    pair: rule; boolean-op                 |
-   |                 |             |                                           |
-   |                 |             | If this rule contains more than one       |
-   |                 |             | condition, a value of ``and`` specifies   |
-   |                 |             | that the rule evaluates to true only if   |
-   |                 |             | all conditions are true, and a value of   |
-   |                 |             | ``or`` specifies that the rule evaluates  |
-   |                 |             | to true if any condition is true.         |
-   +-----------------+-------------+-------------------------------------------+
-
-A ``rule`` element must contain one or more conditions. A condition may be an
-``expression`` element, a ``date_expression`` element, or another ``rule`` element.
+   pair: rule; XML element
```

Comment:
In the future I'd prefer to make "convert to list table" a separate commit when doing these. It makes the side-by-side diff terrible to read.

Reply:
Yeah, usually I do, but this was almost a complete replacement of the chapter so it seemed pointless. Easier to compare the generated documentation